survey/survey-application/src/main/java/me/nalab/survey/application/service/findfeedback/type/FeedbackFindByTypeService.java

```diff
@@ -0,0 +1,83 @@
+package me.nalab.survey.application.service.findfeedback.type;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.RequiredArgsConstructor;
+import me.nalab.survey.application.common.feedback.dto.FeedbackDto;
+import me.nalab.survey.application.common.feedback.mapper.FeedbackDtoMapper;
+import me.nalab.survey.application.common.survey.dto.ChoiceFormQuestionDto;
+import me.nalab.survey.application.common.survey.dto.FormQuestionDtoable;
+import me.nalab.survey.application.common.survey.dto.ShortFormQuestionDto;
+import me.nalab.survey.application.common.survey.dto.SurveyDto;
+import me.nalab.survey.application.common.survey.mapper.SurveyDtoMapper;
+import me.nalab.survey.application.exception.SurveyDoesNotExistException;
+import me.nalab.survey.application.exception.SurveyDoesNotHasTargetException;
+import me.nalab.survey.application.port.in.web.findfeedback.type.FeedbackFindByTypeUseCase;
+import me.nalab.survey.application.port.out.persistence.findfeedback.type.FeedbackFindPort;
+import me.nalab.survey.application.port.out.persistence.findfeedback.type.SurveyFindPort;
+import me.nalab.survey.application.port.out.persistence.findfeedback.type.TargetIdFindPort;
+import me.nalab.survey.domain.feedback.Feedback;
+import me.nalab.survey.domain.survey.Survey;
+
+@Service
+@RequiredArgsConstructor
+public class FeedbackFindByTypeService implements FeedbackFindByTypeUseCase {
+
+	private final SurveyFindPort surveyFindPort;
+	private final FeedbackFindPort feedbackFindPort;
+	private final TargetIdFindPort targetIdFindPort;
+
+	@Transactional
+	@Override
+	public SurveyDto findSurvey(Long surveyId) {
+		Optional<Survey> survey = surveyFindPort.findSurveyById(surveyId);
+		if (survey.isEmpty()) {
+			throw new SurveyDoesNotExistException(surveyId);
+		}
```

Comment:
```suggestion
		Optional<Survey> survey = surveyFindPort.findSurveyById(surveyId)
			.orElseThrow(() -> {throw new SurveyDoesNotExistException(surveyId);});
```

이렇게 줄이는것도 가능해 보이네요!
Optional 사용하면, 가능하면 Optional쪽 API를 사용하자는 주의입니다!